python/pyproject.toml

```diff
@@ -21,7 +21,8 @@ dependencies = [
 
 [project.optional-dependencies]
 srt = ["aiohttp", "fastapi", "hf_transfer", "huggingface_hub", "interegular", "jsonlines",
-       "packaging", "pillow", "psutil", "pydantic", "python-multipart",
+       "matplotlib",
```

Comment:
minimize the dependence. Do not make matplotlib or jsonlines the default dependency because `bench_latency.py` is more like a unit test instead of core functionality.

Reply:
I was thinking the same. Just remove them or list them in a different dependency section?

Reply:
I made a new sglang[dev] dependency.